x-pack/filebeat/module/checkpoint/firewall/manifest.yml

```diff
@@ -9,6 +9,16 @@ var:
     default: 9001
   - name: input
     default: syslog
+  - name: log_level
+    default: 7
+  - name: certfile
```

Comment:
Maybe this could be done link in https://github.com/elastic/beats/blob/fbaeefe7debac749a6c6a962523496aabb0e24f7/x-pack/filebeat/module/okta/system/manifest.yml#L30 and https://github.com/elastic/beats/blob/fbaeefe7debac749a6c6a962523496aabb0e24f7/x-pack/filebeat/module/okta/system/config/input.yml#L39 for consistency. Not a strong opinion on this on my side though.